### DISCUSSION TITLE
CSV Wide and additional billing codes

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#2] [Q&A] [answered]
I have a two part question regarding the additional billing code requirement and how that is to be displayed. 

The scenario is:  Payer1|Plan1 has in their negotiated contract three different Observation rates based on additional billing codes:
- **Rule Out Myocardial Infarction** - Billing codes are Rev 762 And Diagnosis Codes (list of several ICD10 codes) - reimbursement is 10,000 Case Rate

- **Observation with Surgery** - billing codes are Rev 762 and CPT/HCPCS codes (List of several thousand codes) - reimbursement is 0.00 because they get bundled with the surgery reimbursement. 
- **Observation** - billing code Rev 762 (when they don't have the other two code types) - reimbursement is 8,000 Case Rate

**Questions**:
1. When there are multiple codes of the same code type that could be billed in order to get a different negotiated rate, how do we display that?  For reference, Observation with Surgery in the above scenario, there are about 7700 codes that are a mix of CPT and HCPCS codes and I'm unclear how to display that.   

2. We are currently using a format close to the CSV wide template. In the above scenario, Observation is only negotiated this way for Payer1|Product1.  Payer2|Product1 does pay for observation, but it's a case rate for all billing scenarios.  Do I only populate the third line that references just the Rev 762 for Payer2|Product1 or would I also populate the other two scenarios even though it's **not** mentioned that way in Payer2|Product1 negotiated contract, but claims with those billing scenarios **would pay**?   Again using CSV Wide template. 

     
description | code        \| [1] | code        \| [1] \| type | code        \| [2] | code        \| [2] \| type | code        \| [3] | code        \| [3] \| type | modifiers | billing_class | setting | drug_unit_of_measurement | drug_type_of_measurement | standard_charge        \| gross | standard_charge        \| discounted_cash | standard_charge        \| min | standard_charge        \| max | standard_charge        \| Payer1 \| Plan1 | standard_charge        \| Payer1\|Plan1 \| percent | standard_charge        \| Payer1\|Plan1 \| contracting_method | additional_payer_notes        \| Payer1\|Plan1 | standard_charge        \| Payer2 \| Plan1 | standard_charge        \| Payer2\|Plan1 \| percent | standard_charge        \| Payer2\|Plan1 \| contracting_method | additional_payer_notes        \| Payer2\|Plan1 | additional_generic_notes
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Observation | 762 | Rev | XXXXX | CPT/HCPCS |   |   |   |   | Outpatient |   |   | 15,000.00 | 15,000.00 |   |   | 0 |   | Dollar        Amount |   |   |   |   |   |  
Observation | 762 | Rev | xxxxxxx | ICD |   |   |   |   | Outpatient |   |   | 15,000.00 | 15,000.00 |   |   | 10,000.00 |   | Dollar        Amount |   |   |   |   |   |  
Observation | 762 |   |   |   |   |   |   |   | Outpatient |   |   | 15,000.00 | 15,000.00 |   |   | 8,000.00 |   | Dollar        Amount |   |   |   |   |   |

**Comment by shaselton-usds** [comment, marked as answer]
@JaSimps5 Thank you for your questions.

Please refer to the “[CSV Examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples)” on this GitHub site for an example of how to encode standard charge information in this scenario.
